compose/app.yaml

```diff
@@ -58,5 +55,23 @@ services:
       - "8002"
     labels:
       traefik.enable: true
-      traefik.frontend.rule: 'Host: api.wb.local; PathPrefix:/trading, /trading-ui-assets'
+      traefik.frontend.rule: 'Host: www.wb.local; PathPrefix:/peatio/trading, /peatio/trading-ui-assets'
       traefik.port: 8002
+
+  assets:
+    image: nginx
```

Comment:
I'm not sure you will be able to serve static files 
I think we need changes in peatio https://github.com/rubykube/peatio/blob/master/config/environments/production.rb#L30

Reply:
@ysv yes, we need to add one more environment variable to peatio `RAILS_SERVE_STATIC_FILES`.